htlcswitch/link.go

```diff
@@ -3320,6 +3321,14 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg,
 					PaymentHash: pd.RHash,
 				}
 
+				if fwdInfo.NextBlinding != nil {
```

Comment:
Same comment here from the past PR: we can just propagate the option along the whole time if it's returned uniformly.